install-guides/README.md

```diff
@@ -57,36 +26,41 @@ In kubeadm these variables can be set in a file that sits in
  Replace the line KUBELET_EXTRA_ARGS with the following:
 
 ```
-KUBELET_EXTRA_ARGS="\
+KUBELET_EXTRA_ARGS=\
 --cpu-manager-policy=static \
 --kube-reserved=cpu=1,memory=2Gi,ephemeral-storage=1Gi \
```

Comment:
Maybe spare the extra configuration and resources and use ```--reserved-cpus="0"``` instead? 
In a poor system with a lower amount of available CPUs, it might be useful.

Reply:
👍